Tests/Tests/STPFixtures.m

```diff
@@ -10,6 +10,17 @@
 #import "STPTestUtils.h"
 #import "STPEphemeralKey.h"
 
+NSString *const STPJSONKeyCustomer = @"Customer";
```

Comment:
nit: not sure about `STPJSONKey` as a name, since it kinda looks like these are constants from the SDK. Maybe something like `STPTestJSON`?

Reply:
Good idea. I wasn't sure what to call them. As I was working I renamed all of them 3-4 times before pushing this :P